index.bs

```diff
@@ -490,6 +490,20 @@ For better accessibility, a web-based presentation application provides
 automatic image captioning by running a machine learning model such as
 [[im2txt]] which predicts explanatory words of the presentation slides.
 
+### Text-to-image ### {#usecase-text-to-image}
+
+Images are a core part of modern web experiences. An ability to in a
```

Comment:
This sentence is awkward to read ("split infinitive"). Suggested rewording:

> An ability to generate images based on text input in a privacy-preserving manner enables...

or:

> An ability to generate images in a privacy-preserving manner based on text input enables...

Reply:
Thanks! This _felt_ somehow wrong when I wrote it, but just couldn't figure out why or how to improve :)

Fixed by https://github.com/webmachinelearning/webnn/pull/507/commits/1d0f5c04967fdd4cec6c3da7f942d8eeb1265e1f